lib/ohai/plugins/windows/features.rb

```diff
@@ -0,0 +1,20 @@
+Ohai.plugin(:Features) do
+  provides "installed_features"
+  collect_data(:default) do
```

Comment:
It's best practice to restrict plugins which collect platform-specific data to the specific platform. You can tell Ohai to collect this data only on Windows by replacing `:default` (runs on all platforms) with `:windows`.